extensions/README.md

```diff
@@ -6,12 +6,12 @@
 * [KHR_materials_pbrSpecularGlossiness](2.0/Khronos/KHR_materials_pbrSpecularGlossiness/README.md)
 * [KHR_materials_unlit](2.0/Khronos/KHR_materials_unlit/README.md)
 * [KHR_draco_mesh_compression](2.0/Khronos/KHR_draco_mesh_compression/README.md)
+* [KHR_techniques_webgl](2.0/Khronos/KHR_techniques_webgl/README.md)
```

Comment:
Can we mark this as draft here and in the main extension spec and then merge this?